api/services/elastigroup/azure/spotVms/paths/groupVmDeploymentDetails.yaml

```diff
@@ -8,7 +8,7 @@ get:
   parameters:
     - $ref: "../../../../../commons/parameters/accountId.yaml"
     - $ref: "../../../../../commons/parameters/groupId.yaml"
-    - in: path
```

Comment:
wrong change.
URL is https://api.spotinst.io/azure/compute/group/{groupId}/deployment/{**deploymentId**}/details

Reply:
You are correct, corrected.